capstone-frontend/src/BookSearchTile.test.js

```diff
@@ -42,16 +42,22 @@ test('renders book search tile test', () => {
   const bookListButton = document.getElementById("dropdown-list-add");
   expect(bookListButton).toBeInTheDocument();
   expect(bookListButton.innerHTML).toEqual("Add to List");
-  
 
+  await act(async () => {bookListButton.click()});
+  const dropdownItems = document.getElementsByClassName("dropdown-item");
+  expect(dropdownItems.length).toEqual(3);
+  expect(dropdownItems[0].innerHTML).toEqual("<span>Best Books</span>");
```

Comment:
The fact that the text is wrapped in a `span` is really an implementation detail that would be best not to test directly. See if there's a way you can assert on only the text content here.

Reply:
I changed this to `toMatch(/Create New List/)` with the innerHTML, because I try to get .innerText it gives me nothing. In this way, I don't care about the span, just about the Create New List text.